package.json

```diff
@@ -7,7 +7,8 @@
     "build": "next build",
     "start": "next start",
     "add-git-hooks": "husky install",
-    "postinstall": "is-ci || husky install"
+    "postinstall": "is-ci || husky install",
+    "lint": "yarn eslint . && yarn stylelint **/*.css"
```

Comment:
Maybe break it into two pieces, one for JS linting, another for style linting. Then this command can be invoked ergonomically.

Reply:
Fixed.